.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### General (all pull requests)
+
+- [ ] I have read the [contribution guide](https://osu.ppy.sh/wiki/en/osu%21_wiki_Contribution_Guide)
```

Comment:
Yeah I don't know about this one item especially. There is nothing stopping people from just checking this off without actually reading. Yeah it would be a crappy move on their behalf but this is mostly performative, and I'd get tired of checking this off every time *very* fast if I were PRing to wiki more often.

Sad thing is that generally speaking random new contributors don't (*or won't*) read and that's from speaking from experience over at the lazer repo. You can try issue templates, contributing guidelines - if it's long or technical enough people will just ignore anyway.

Reply:
sad but true, this is useless the way I use it (as a requirement)